Makefile

```diff
@@ -6,9 +6,15 @@ all: install
 fmt:
 	go fmt ./...
 
+# REBUILD touches all of the build-dependent source files, forcing them to be
+# rebuilt. This is necessary because the go tool is not smart enough to trigger
+# a rebuild when build tags have been changed.
+REBUILD:
+	@touch consensus/build*.go
```

Comment:
positioning seems weird, but I guess it makes sense. The name is weird too because you aren't actually rebuilding anything you're just touching a few files.